packages/styles/scss/components/content-item-horizontal/_content-item-horizontal.scss

```
@@ -37,7 +37,7 @@
   }
 
   :host(#{$dds-prefix}-content-item-horizontal) ::slotted([slot='heading']),
-  :host(#{$dds-prefix}-content-item-horizontal) .#{$prefix}--content-item__copy,
+  :host(#{$dds-prefix}-content-item-horizontal) ::slotted([slot='copy']),
```

Probably we can keep using the default slot for copy content in this way...?

```suggestion
  :host(#{$dds-prefix}-content-item-horizontal) ::slotted(:not([slot])),
```